src/index.js

```diff
@@ -1,33 +1,55 @@
+import bodyParser from "body-parser";
+import cookieParser from "cookie-parser";
+import cors from "cors";
 import dotenv from "dotenv";
-dotenv.config();
 import express, { static as staticExpress } from "express";
 import mongoose from "mongoose";
-import cookieParser from "cookie-parser";
+import path, { join } from "path";
 import swaggerJSDoc from "swagger-jsdoc";
 import swaggerUi from "swagger-ui-express";
-import path, { join } from "path";
-import cors from "cors";
-const __dirname = path.resolve();
-import { optionsToCustomizeSwagger, swaggerOptions } from "./base.js";
-import { connectDB } from "./base.js";
+import {
+  connectDB,
+  optionsToCustomizeSwagger,
+  swaggerOptions,
+} from "./base.js";
 import { router as IndexRouter } from "./indexApp/routes.js";
+import { IndexView } from "./indexApp/views.js";
+import { errLogger, logger } from "./config/utils.js";
+
+dotenv.config();
+connectDB();
+const __dirname = path.resolve();
+const swaggerSpec = swaggerJSDoc(swaggerOptions);
+const apiRoute = process.env.API_BASE || "/api/v1/";
 
 const app = express();
+app.use(logger);
 app.use(cors());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
```

Comment:
I faced an issue ever since I started to use **swagger**. When I try to use the express version, they send me an error. 
### Attached here for reference.
The error is that express is not **shipped with body-parser**


![image](https://user-images.githubusercontent.com/28519395/150040326-1ccdc231-8b2c-47fc-877d-28dc651a32f4.png)
![image](https://user-images.githubusercontent.com/28519395/150040338-70ce7c95-c81d-4bbc-82de-ea021514500c.png)